synapse/tests/test_cmds_cortex.py

```diff
@@ -150,6 +197,18 @@ class CmdCoreTest(s_t_utils.SynTest):
             await cmdr.runCmdLine('storm test:str ->')
             # TODO: figure out how to evaluate whether these made it to the screen
 
+            # No more color related testing after this block
+            s_cli.ColorsEnabled = False
+
+        # Attempt to use a closed proxy - this fails as soon
+        # as we try to execut the command
+        await asyncio.sleep(0.5)
```

Comment:
Is there no real event/object we can wait on here?  Arbitrary sleeps in test code usually bite us later...

Reply:
could just use waitfini on the proxy.